auth/migrations/0000_init.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE IF NOT EXISTS users (
+  account_name TEXT PRIMARY KEY,
+  secret TEXT UNIQUE,
```

Comment:
Is `secret` the old `key`?

Reply:
I was a bit confused about if this would be the same as gateway and I was thinking we would use an encryption secret here, but the issue did say it would be the same :smile: I'll change it to be the same for now.